profiler/test/Datadog.Profiler.Native.Tests/LinuxStackFramesCollectorTest.cpp

```diff
@@ -315,7 +315,7 @@ TEST_F(LinuxStackFramesCollectorFixture, CheckCollectionAbortIfInPthreadCreateCa
     auto collector = LinuxStackFramesCollector(signalManager);
 
     auto threadInfo = ManagedThreadInfo((ThreadID)0);
-    threadInfo.SetOsInfo((DWORD)GetWorkerThreadId(), (HANDLE)0);
+    threadInfo.SetOsInfo((DWORD)GetWorkerThreadId(), (HANDLE)nullptr);
```

Comment:
I'm not sure HANDLE should be seen as a pointer: most of the time it is an offset in a table so 0 makes more sense than nullptr